### ISSUE TITLE
Auto upload by selecting media type

### ISSUE THREAD

**Issue by CrazyShipOne** [nextcloud/ios#2284]
Hi,

Really appreciated for such amazing app! It does easy my life for a long time.

I use it to backup all my photo by auto upload. The only thing it matters is that screenshots are also backed up, which I do not want.

So is there any possibility to add a dropdown to select media type for auto upload, just like viewing different types of items in IOS's album? Thus I can only back up photos and videos shoot by camera.

Thanks!

**Comment by kdjordjev** [comment]
I second the request.
I was an Android user and the Android app had the option to select different media types - Photos, Video, Screenshots, What'sApp Attachment, etc ....  For each media type you can select to AuitoSync or not and folder destination.

This was very handy to me, since i synced the Videos and Photos to different locations. 
I would really appreciate if we can recreate the same architecture in iOS 

Tx

**Comment by jth134** [comment]
#2377 has a similar request and would be beneficial to select/deselect PNG uploads to prevent screenshot uploads

**Comment by shubhpsd** [comment]
Yes, this is much needed, I use nextcloud to sync my obsidian notes, it would be really helpful if they can add a automatic sync let you choose any folder/file from the files menu pop-up or something, this would be of huge help and is the last piece of my syncing puzzle.
